services/apis/etl/etlTasks.js

```diff
@@ -9,6 +9,25 @@ const bash = new Bash();
  * ETL Tasks
  */
 module.exports = {
+  // TODO docstring
+  cleanUpIndices() {
```

Comment:
we should not delete all indices, should leave the ones that are aliased ... otherwise the alias itself might get deleted in the cleanup and prevent guppy from coming up on subsequent runs

Reply:
I meant aliases might get cleaned up by the cloud-auto's garbage collection or AWS' snapshot process if they are not linked to an index, I am not 100% sure how it happened but have had cases where the alias itself goes away and I had to manually recreate them

Reply:
After the gitops-qa PR is merged, it should no longer matter though if those unlinked aliases get cleaned up because that PR specifically makes Guppy not use aliases associated with the names in `etlMapping.yaml`.